science/proxmark3-iceman/Portfile

```
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                1.0
+PortGroup                 github 1.0
+
+name                      proxmark3-iceman
+version                   4.13441
+revision                  0
+license                   GPL-2
+categories                science comms
+platforms                 darwin macosx
+
+description               RRG/Iceman version of proxmark3 RFID Instrument Client/FW
+
+long_description          ${description}. By default, the repository is built for PM3RDV4. \
+                          To build for older hardware releases, please install the package \
+                          with the "pm3generic" variant.
```

If you want these quotes preserved in the output of `port info` you have to escape them.
```suggestion
                          with the \"pm3generic\" variant.
```